railties/lib/rails/commands/console/console_command.rb

```diff
@@ -12,6 +12,53 @@ def filter_backtrace(bt)
       end
     end
 
+    class IRBConsole
+      def initialize
+        require "irb"
+        require "irb/completion"
+        IRB::WorkSpace.prepend(BacktraceCleaner)
+
+        if !Rails.env.local?
+          # Use env var here so users can override them with env var too
+          ENV["IRB_USE_AUTOCOMPLETE"] ||= "false"
+        end
+      end
+
+      def name
+        "IRB"
+      end
+
+      def start
+        IRB.setup(nil)
+
+        env = colorized_env
+
+        IRB.conf[:PROMPT][:RAILS_PROMPT] = {
+          PROMPT_I: "#{env}:%03n> ",
+          PROMPT_S: "#{env}:%03n%l ",
+          PROMPT_C: "#{env}:%03n* ",
```

Comment:
I don't think we need line numbers

Reply:
It's removed in https://github.com/rails/rails/pull/50814

Reply:
I see ![Screenshot 2024-01-26 at 15 24 54](https://github.com/rails/rails/assets/58794487/64671c15-b68e-4123-978e-f62ef4c4063f)